phpunit.xml.dist

```diff
@@ -8,16 +8,17 @@
          beStrictAboutOutputDuringTests="true"
          beStrictAboutTestsThatDoNotTestAnything="true"
          beStrictAboutTodoAnnotatedTests="true"
+         bootstrap="vendor/autoload.php"
```

Comment:
I'm sure this is need it but I'm not sure why. Or rather, I'm not sure why it worked before. It's as if phpunit assumed psr 0